### PR TITLE
Fix bug with fastclick and dayclick on iOS #260

### PR DESCRIPTION
**Pull request by mlumbroso**
Dayclicks not working when using Fastclick, I fixed it the same way as Pikaday did : https://github.com/dbushell/Pikaday/issues/101

### CODE REVIEW

File: dist/fullcalendar.js [hunk 1]
```diff
@@ -3296,6 +3296,16 @@ var Grid = fc.Grid = RowRenderer.extend({
 		// attach a handler to the grid's root element.
 		// jQuery will take care of unregistering them when removeElement gets called.
 		el.on('mousedown', function(ev) {
```
Comment (arshaw) [inline dist/fullcalendar.js]: if the handlers is exactly the same, couldn't we do `el.on('mousedown touchstart', ...` ?

**Comment by arshaw** [comment]
would be great if the files in `src/` were modified instead

**Comment by arshaw** [comment]
closing this PR but will reference when addressing #260

**Comment by gmunumel** [comment]
Still with this fix I can't get it work. I'm using iOS9

**Pull request closed without merging**